src/api.js

```diff
@@ -9,26 +8,25 @@ const newRoomEndpoint =
  * in README.md
  *
  * See https://docs.daily.co/reference#create-room for more information on how
- * to use the Daily REST API to create rooms and what options are available. 
+ * to use the Daily REST API to create rooms and what options are available.
  */
 async function createRoom() {
-
   const exp = Math.round(Date.now() / 1000) + 60 * 30;
   const options = {
     properties: {
       exp: exp,
     },
   };
   let response = await fetch(newRoomEndpoint, {
-    method: "POST",
-    body: JSON.stringify(options),
-    mode: 'cors',
-  }),
+      method: 'POST',
+      body: JSON.stringify(options),
+      mode: 'cors',
+    }),
     room = await response.json();
   return room;
 
   // Comment out the above and uncomment the below, using your own URL
-  // return { url: "https://your-domain.daily.co/hello" };
+  // return { url: 'https://liza.staging.daily.co/mutest' };
```

Comment:
Probably want to switch this back to `https://your-domain.daily.co/hello`!

Reply:
Oops yes, great catch >.<